docs/src/examples.md

```diff
@@ -2,543 +2,180 @@
-
-#### Equations and discretizations
-
-#### Mass
-
-Follows the continuity equation
-```math
-\begin{equation}
-  \frac{\partial}{\partial t} \rho = - \nabla \cdot(\rho \boldsymbol{u}) .
-\label{eq:3d-box-advection-lim-continuity}
-\end{equation}
-```
 
-This is discretized using the following
+These are discretized using the following:
 ```math
-\begin{equation}
-  \frac{\partial}{\partial t} \rho \approx - D_h[ \rho (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))] - D^c_v[I^f(\rho \boldsymbol{u}_h)) + I^f(\rho) \boldsymbol{u}_v)] .
-\label{eq:3d-box-advection-lim-discrete-continuity}
-\end{equation}
-```
-
-#### Tracers
-
-For the tracer concentration per unit mass ``q``, the tracer density (scalar) ``\rho q`` follows the advection/transport equation
-
-```math
-\begin{equation}
-  \frac{\partial}{\partial t} \rho q = - \nabla \cdot(\rho q \boldsymbol{u})  + g(\rho, q).
-\label{eq:3d-box-advection-lim-tracers}
-\end{equation}
-```
-
-This is discretized using the following
-```math
-\begin{equation}
-\frac{\partial}{\partial t} \rho q \approx
-- D_h[ \rho q (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))]
-- D^c_v\left[I^f(\rho q) U^f\left(I^f(\boldsymbol{u}_h) + \boldsymbol{u}_v, \frac{\rho q}{\rho} \right) \right] + g(\rho, q),
-\label{eq:3d-box-advection-lim-discrete-tracers}
-\end{equation}
+\begin{align}
+\frac{\partial u}{\partial t} &\approx D_{f2c}\left(\nu G_{c2f}(u)\right) + f(v - v_g) - A(w, u) \\
+\frac{\partial v}{\partial t} &\approx D_{f2c}\left(\nu G_{c2f}(v)\right) - f(u - u_g) - A(w, v)
+\end{align}
 ```
-where ``g(\rho, q) = - \nu_4 [\nabla^4_h (\rho q)]`` represents the horizontal hyperdiffusion operator, with ``\nu_4`` (measured in m^4/s) the hyperviscosity constant coefficient.
-
-Currently tracers are only treated explicitly in the time discretization.
-
 
 #### Prognostic variables
 
-* ``\rho``: _density_ measured in kg/m³. This is discretized at cell centers.
-* ``\boldsymbol{u}`` _velocity_, a vector measured in m/s. This is discretized via ``\boldsymbol{u} = \boldsymbol{u}_h + \boldsymbol{u}_v`` where
-  - ``\boldsymbol{u}_h = u_1 \boldsymbol{e}^1 + u_2 \boldsymbol{e}^2`` is the projection onto horizontal covariant components (covariance here means with respect to the reference element), stored at cell centers.
-  - ``\boldsymbol{u}_v = u_3 \boldsymbol{e}^3`` is the projection onto the vertical covariant components, stored at cell faces.
-* ``\rho q``: the tracer density scalar, where ``q`` is the tracer concentration per unit mass, is stored at cell centers.
-
-#### Operators
-
-We make use of the following operators
+* `u`: _horizontal velocity (east-west component)_ measured in m/s.
```

Comment:
Here is a suggestion using terminology that is more familiar to this community (please see this [Wiki](https://en.wikipedia.org/wiki/Zonal_and_meridional_flow) reference)

```suggestion
* `u`: _horizontal velocity (zonal or east-west component)_ measured in m/s.
```